resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/config.scala

```diff
@@ -0,0 +1,175 @@
+          | API server. This file should be located on the submitting
+          | machine's disk.
+        """.stripMargin)
+      .stringConf
+      .createOptional
+
+  private[spark] val KUBERNETES_CLIENT_CERT_FILE =
+    ConfigBuilder("spark.kubernetes.submit.clientCertFile")
+      .doc("""
+          | Client cert file for authenticating against the
+          | Kubernetes API server. This file should be located on
+          | the submitting machine's disk.
+        """.stripMargin)
+      .stringConf
+      .createOptional
+
+  private[spark] val KUBERNETES_SERVICE_ACCOUNT_NAME =
+    ConfigBuilder("spark.kubernetes.submit.serviceAccountName")
+      .doc("""
+          | Service account that is used when running the driver pod.
+          | The driver pod uses this service account when requesting
+          | executor pods from the API server.
+        """.stripMargin)
+      .stringConf
+      .createWithDefault("default")
+
+  private[spark] val KUBERNETES_DRIVER_UPLOAD_JARS =
+    ConfigBuilder("spark.kubernetes.driver.uploads.jars")
+      .doc("""
+          | Comma-separated list of jars to sent to the driver and
+          | all executors when submitting the application in cluster
+          | mode.
+        """.stripMargin)
+      .stringConf
+      .createOptional
+
+  // Note that while we set a default for this in practice, it's
+  // dynamically determined based on the executor memory.
+  private[spark] val KUBERNETES_EXECUTOR_MEMORY_OVERHEAD =
+    ConfigBuilder("spark.kubernetes.executor.memoryOverhead")
+      .doc("""
+          | The amount of off-heap memory (in megabytes) to be
+          | allocated per executor. This is memory that accounts for
+          | things like VM overheads, interned strings, other native
+          | overheads, etc. This tends to grow with the executor size
+          | (typically 6-10%).
+        """.stripMargin)
+      .stringConf
+      .createOptional
+
+  private[spark] val KUBERNETES_DRIVER_LABELS =
+    ConfigBuilder("spark.kubernetes.driver.labels")
+      .doc("""
+          | Custom labels that will be added to the driver pod.
+          | This should be a comma-separated list of label key-value
+          | pairs, where each label is in the format key=value.
+        """.stripMargin)
+      .stringConf
+      .createOptional
+
+  private[spark] val KUBERNETES_DRIVER_SUBMIT_TIMEOUT =
+    ConfigBuilder("spark.kubernetes.driverSubmitTimeout")
+      .doc("""
+          | Time to wait for the driver pod to be initially ready
+          | before aborting the job.
+        """.stripMargin)
+      .timeConf(TimeUnit.SECONDS)
+      .createWithDefault(60L)
+
+  private[spark] val KUBERNETES_DRIVER_SUBMIT_KEYSTORE =
+    ConfigBuilder("spark.ssl.kubernetes.submit.keyStore")
+      .doc("""
+          | KeyStore file for the driver submission server listening
+          | on SSL. Can be pre-mounted on the driver container
+          | or uploaded from the submitting client.
```

Comment:
I don't think these are published by a script or something by any means - this is probably solely developer documentation. I think going without `container://` is fine in that context.